x/bank/keeper/msg_server.go

```diff
@@ -102,7 +102,7 @@ func (k msgServer) MultiSend(goCtx context.Context, msg *types.MsgMultiSend) (*t
 
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	// NOTE: totalIn == totalOut should already have been checked
+	// NOTE: total In == total Out should already have been checked
```

Comment:
hmm not valid fix, but good to keep anyway

Reply:
added totalIn and totalOut on ignorelist